Module.Build.json

```diff
@@ -0,0 +1,23 @@
+{
+    "Resources": {
+		"NoComponents": true,
+		"CoreServicesDlls":[
+			"./Decisions.Microsoft365.Common/obj/Decisions.Microsoft365.Common.dll"
+		],
+		"ReferenceDlls":
+		[
+			"./Decisions.Microsoft365.Common/obj/Microsoft.Graph.dll",
```

Comment:
Make sure to take these out of the Exchange module. They will conflict if they get copied from both.
This is why we want to push those MS Model dependencies out of our step classes. Easier to manage the dependency graph (theoretically) if they only come from one place.